{ 'test': { 'left_arm': [],
            'left_leg': [ { 'duration': 2.029,
                            'name': u'high_step_5_left',
                            'positions': [ 0.01,
                                           -0.0392,
                                           -1.4572,
                                           1.8067,
                                           0.528,
                                           0.0156],
                            'starttime': 0.0}],
            'neck': [],
            'right_arm': [],
            'right_leg': [ { 'duration': 2.029,
                             'name': u'high_step_5_right',
                             'positions': [ 0.02,
                                            -0.0437,
                                            -0.1025,
                                            -0.0467,
                                            -0.2793,
                                            0.0478],
                             'starttime': 0.0}],
            'torso': []}}
